app/src/forms/attestations/minesoperatorscreening/models/form.js

```diff
@@ -106,6 +106,23 @@ class Attestation extends AttestationModels.AttestationTransportation {
     return PREFIX;
   }
 
+  static get jsonSchema() {
+    const schemaProperties = {...AttestationModels.AttestationTransportationSchema.properties};
+    // add new properties
+    schemaProperties.transportationFaceCovering = {type: 'boolean'};
+    schemaProperties. distancingStayInAccom = {type: 'boolean'};
+    schemaProperties.trainingDailyChecks = {type: 'boolean'};
```

Comment:
Not really sure what this is for? These are fields in the  submission_attestation table so should be defined in the same place as all the other ones. Any reason the 3 new fields are singled out in this new method?

Reply:
Yes, it does look a bit strange but it seemed to follow the convention already there. 

Any changes to the models that arent common to all forms are in app/src/forms/<form>/models/form.js
the agriseafood and mines forms have some additional properties by extending the AttestationTransportation model as well as adding the sleeping related ones.

If i add these three new properties to the 'base' attestation model, then they are required for all 3 forms.

i think youre right, i might be able to do this a cleaner way

Reply:
👍